### DISCUSSION TITLE
:Telescope repo cached_list was unsccessful. Exit code: 1

### DISCUSSION THREAD

**Discussion by fz18** [cljoly/telescope-git-repo.nvim#67] [Q&A]
when neovim checkhealth: telescope._extensions.repo: require("telescope._extensions.repo.health").check()

ERROR: Telescope repo cached_list was unsccessful. Exit code: 1
locate -r /\.git$ -l 2
locate: illegal option -- rnt
usage: locate [-0Scims] [-l limit] [-d database] pattern ...

my os is mac
System Version: macOS 13.5 
Kernel Version: Darwin 22.6.0

**Comment by cljoly** [comment]
It looks like you might not be using gnu locate. See also https://github.com/cljoly/telescope-repo.nvim#notes-for-macos
